hue/verify_hue_running.py

```diff
@@ -52,12 +61,13 @@ def main():
     hue = Hue(BASE, PORT)
     is_main_master = hue.is_main_master()
     if is_main_master:
-        code = hue.get_response_code()
+        code = hue.get_response_code(retries=100)
         assert code is not None, "NOK - Failed to get a status code"
```

Comment:
Retry 100 times? seems to be too large.

Reply:
With 3 seconds delay, it just 5 minutes.